sig-node/annual-report-2021.md

```diff
@@ -0,0 +1,67 @@
+* What programs do you participate in for new contributors?
+  * KubeCon updates with “how to get involved”, beginning at KubeCon EU 2021
+  * Improving documentation and process so new individuals can join, see e.g. [Node Project Board](https://github.com/orgs/kubernetes/projects/49), [Node PR Triage Guidelines](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-node/triage.md)
+  * Mentoring group for underrepresented individuals to become reviewers, first cohort (July-Dec. 2020) run by @dims and second cohort (Jan-Mar. 2021) run by @ehashman
+  * Next cohort leader/participants TBD  
+* Does the group have contributors from multiple companies/affiliations? Can end users/companies contribute in some way that
+they currently are not?
+  * The group has contributors from multiple companies/affiliations.
+  * [39 companies made 1+](https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=y&var-metric=contributions&var-repogroup_name=SIG%20Node&var-companies=All) contributions over the last year (70 if all activity is counted).
+  * Deprecating of dockershim is a big project that will require many end user to make a step to migrate and some companies who are not active on SIG Node, namely monitoring vendors, to help end users with this migration.
+
+## Current initiatives and project health
+
+* [x] Please include links to KEPs and other supporting information that will be beneficial to multiple types of community members.
+* What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? Currently underway?
+What are some of the longer tail projects that your group is working on?
+  * Better tracking of [PRs setup](https://github.com/orgs/kubernetes/projects/49) and[Node PR Triage Guide](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-node/triage.md) by @ehashaman
+  * cgroups v2
+  * [Graduating features](https://docs.google.com/document/d/1U10J0WwgWXkdYrqWGGvO8iH2HKeerQAlygnqgDgWv4E/edit#)
+  * Topology manager / device alignment
+* Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?
+  * TODO: 2020 KEPs summary
+  * 1.18: (https://github.com/kubernetes/enhancements/issues?q=is%3Aissue+milestone%3Av1.18+label%3Asig%2Fnode)
+  * 1.19: (https://github.com/kubernetes/enhancements/issues?q=is%3Aissue+milestone%3Av1.19+label%3Asig%2Fnode+)
+  * 1.20: (https://github.com/kubernetes/enhancements/issues?q=is%3Aissue+milestone%3Av1.20+label%3Asig%2Fnode+)
+* What areas and/or subprojects does the group need the most help with?
+  * TODO
+* What's the average open days of a PR and Issue in your group? / what metrics does your group care about and/or measure?
```

Comment:
btw, devstats charts seems to be broken:

- https://k8s.devstats.cncf.io/d/44/pr-time-to-approve-and-merge?orgId=1&var-period=y&var-repogroup_name=SIG%20Node&var-apichange=All&var-size_name=All&var-kind_name=All
- https://k8s.devstats.cncf.io/d/14/pr-time-to-engagment?orgId=1&var-period=y&var-repogroup_name=SIG%20Node&var-kind_name=All

CC: @lukaszgryglicki

Reply:
They're fixed now.

Reply:
@lukaszgryglicki sorry, just looked again on this. This chart https://k8s.devstats.cncf.io/d/44/pr-time-to-approve-and-merge?orgId=1&var-period=y&var-repogroup_name=SIG%20Node&var-apichange=All&var-size_name=All&var-kind_name=All is still showing a single bar:

![image](https://user-images.githubusercontent.com/9950081/114252338-750f2d00-9959-11eb-93f5-8c7706b0eaff.png)

Switching to month gives some breakdown:

![image](https://user-images.githubusercontent.com/9950081/114252360-8d7f4780-9959-11eb-8c2e-db8aae7a3961.png)

But it would be great to get the chart over a year. Really curious if we made progress